uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/WorkPatternTest.php

```diff
@@ -642,4 +643,70 @@ private function getWorkDayTypeOptionsArray() {
 
     return $options;
   }
+
+  public function testItDoesNotEnqueueTaskToUpdatePublicHolidayLeaveRequestsWhenANewWorkPatternIsCreatedAndNotSetToDefault() {
+    WorkPatternFabricator::fabricate(['is_default' => 0]);
+    $queue = PublicHolidayLeaveRequestUpdatesQueue::getQueue();
+    $this->assertEquals(0, $queue->numberOfItems());
+  }
+
+  public function testItEnqueueTaskToUpdatePublicHolidayLeaveRequestsWhenANewWorkPatternIsCreatedSetToDefault() {
+    $workPattern = WorkPatternFabricator::fabricate(['is_default' => 1]);
+    $queue = PublicHolidayLeaveRequestUpdatesQueue::getQueue();
+    $this->assertEquals(1, $queue->numberOfItems());
```

Comment:
You're using this same thing on all the new test methods you've added (instantiate queue, check number of items, claim task, check that is the expected class). Can you please move it a custom assert method and just reuse it on the tests?